entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntityOperationServiceTests.kt

```diff
@@ -249,29 +224,32 @@ class EntityOperationServiceTests {
         every { neo4jRepository.deleteEntityAttributes(firstEntityURI) } returns mockk()
         every {
             entityService.appendEntityAttributes(eq(firstEntityURI), any(), any())
-        } returns UpdateResult(
+        } returns UpdateAttributesResult(
             emptyList(),
             emptyList()
         )
         every { neo4jRepository.deleteEntityAttributes(secondEntityURI) } returns mockk()
         every {
             entityService.appendEntityAttributes(eq(secondEntityURI), any(), any())
-        } returns UpdateResult(
+        } returns UpdateAttributesResult(
             emptyList(),
             listOf(
-                NotUpdatedDetails("attribute#1", "reason"),
-                NotUpdatedDetails("attribute#2", "reason")
+                NotUpdatedAttributeDetails("attribute#1", "reason"),
+                NotUpdatedAttributeDetails("attribute#2", "reason")
             )
         )
 
         val batchOperationResult = entityOperationService.replace(listOf(firstEntity, secondEntity))
 
-        assertEquals(listOf(firstEntityURI), batchOperationResult.success)
+        assertEquals(
+            mutableListOf(BatchEntityReplaceSuccess(firstEntityURI, emptyList())),
+            batchOperationResult.success
+        )
         assertEquals(
             listOf(
                 BatchEntityError(
                     secondEntityURI,
-                    arrayListOf("attribute#1 : reason, attribute#2 : reason")
+                    arrayListOf("1 : reason, 2 : reason")
```

Comment:
I updated the returned attributes in BatchEntityError to reduced format (in both replace and update cases).